src/ngraph/frontend/onnx_import/utils/common.hpp

```diff
@@ -68,53 +68,6 @@ namespace ngraph
                 return range;
             }
 
-            /// \brief      Handle out of range axis.
-            ///
-            /// \param[in]  node         The node with requested axis.
-            /// \param[in]  axis         The requested axis value.
-            /// \param[in]  tensor_rank  The corresponding tensor rank.
-            ///
-            /// \return    Checking if axis is in range [-tensor_rank, tensor_rank-1], otherwise
-            /// returns error.
-            ///            If negative axis, it counts from the last to the first axis, by adding
-            ///            tensor_rank to axis.
-            ///
-            std::size_t validate_axis(const ngraph::onnx_import::Node& node,
-                                      std::int64_t axis,
-                                      std::int64_t tensor_rank);
-
-            /// \brief      Handle out of range axis.
-            ///
-            /// \param[in]  node            The node with requested axis.
-            /// \param[in]  axis            The requested axis value.
-            /// \param[in]  tensor_rank     The corresponding tensor rank.
-            /// \param[in]  axis_range_min  The min value of accepted range for axis.
-            /// \param[in]  axis_range_max  The max value of accepted range for axis.
-            ///
-            /// \return     Checking if axis is in range [axis_range_min, axis_range_max], otherwise
-            /// returns error.
-            ////            If negative axis, it counts from the last to the first axis, by adding
-            /// tensor_rank to axis.
-            ///
-            std::size_t validate_axis(const ngraph::onnx_import::Node& node,
-                                      std::int64_t axis,
-                                      std::int64_t tensor_rank,
-                                      std::int64_t axis_range_min,
-                                      std::int64_t axis_range_max);
-
-            /// \brief      Handle out of range axes in vector.
-            ///
-            /// \param[in]  node         The node with requested axes.
-            /// \param[in]  axes         The requested vector of axes.
-            /// \param[in]  tensor_rank  The corresponding tensor rank.
-            ///
-            /// \return     If any negative axis in vector, it counts from the last to the first
-            /// axis, by adding tensor_rank to axis.
-            ///
-            std::vector<std::size_t> validate_axes(const ngraph::onnx_import::Node& node,
-                                                   std::vector<std::int64_t> axes,
```

Comment:
Please pass this parameter by constant reference.

Reply:
Done